test/e2e/aks.go

```diff
@@ -202,7 +202,7 @@ func WaitForControlPlaneMachinesToExist(ctx context.Context, input WaitForContro
 				ownerMachinePool := &clusterv1exp.MachinePool{}
 				if err := input.Getter.Get(ctx, types.NamespacedName{Namespace: input.Namespace, Name: ref.Name},
 					ownerMachinePool); err != nil {
-					Logf("Failed to get machinePool: %+v", err)
+					LogWarningf("Failed to get machinePool: %+v", err)
```

Comment:
this seems like an actual failure and it's not in a retry block, why log it as a warning?

Reply:
There is no existent error equivalent of `Logf` Do we want to create one?

Reply:
I see, Logf is actually Info which is even less acurrate. I thought we were decreasing severity.